src/index.js

```diff
@@ -12,14 +14,6 @@ const year = date.getFullYear();
 
 console.log("\nMonitor Utility \nVersion: " + version);
 
-console.log(`
---------------------------------------------------
```

Comment:
This is still true for exporting projects only, right?

Reply:
That is correct. I intended to move it lower and forgot. Doing that now.

Reply:
Correction. It was moved lower.  This way it only shows if you are exporting projects (not when exporting issues).